examples/misc_controls_orbit.html

```diff
@@ -51,6 +51,7 @@
 				// controls
 
 				controls = new OrbitControls( camera, renderer.domElement );
+				controls.listenKeyEvents( window );
```

Comment:
Users copy these examples. I'd add a comment to make it clear that this is optional.
```js
controls.listenKeyEvents( window ); // optional
```

Reply:
Maybe rename to `.listenForKeyEvents()` -- or `.listenToKeyEvents()`.

Or just `.enableKeyEvents()`.

Reply:
Renaming to `.listenToKeyEvents()` sounds good to me 👍

@Mugen87 do you mind updating the PR?

Reply:
Updated the PR 🙌 .